plugins/kubernetes/app/models/concerns/kubernetes/deploy_yaml.rb

```diff
@@ -0,0 +1,61 @@
+module Kubernetes
+  module DeployYaml
+    def deployment_hash
+      @deployment_yaml ||=
+        if replication_controller_doc.present?
+          JSON.parse(replication_controller_doc).with_indifferent_access
```

Comment:
In which case can we end up in this situation? When scaling-up / scaling-down an existing release?